api/nodes/views.py

```diff
@@ -125,16 +125,78 @@ def fetch_from_waterbutler(self):
+    field](/v2/#osf-node-categories) that includes the option of project. The categorization essentially determines
+    which icon is displayed by the node in the front-end UI and helps with search organization. Top-level nodes may have
+    a category other than project, and children nodes may have a category of project.
+
+    ##Node Attributes
+
+    <!--- Copied Attributes from NodeDetails -->
+
+    OSF Node entities have the "nodes" `type`.
+
+        name           type               description
+        ---------------------------------------------------------------------------------
+        title          string             title of project or component
+        description    string             description of the node
+        category       string             node category, must be one of the allowed values
+        date_created   iso8601 timestamp  timestamp that the node was created
+        date_modified  iso8601 timestamp  timestamp when the node was last updated
+        tags           array of strings   list of tags that describe the node
+        registration   boolean            has this project been registered?
+        collection     boolean            is this node a collection of other nodes?
+        dashboard      boolean            is this node visible on the user dashboard?
+        public         boolean            has this node been made publicly-visible?
+
+    ##Links
+
+    See the [JSON-API spec regarding pagination](http://jsonapi.org/format/1.0/#fetching-pagination).
+
+    ##Actions
+
+    ###Creating New Nodes
+
+        Method:        POST
+        URL:           links.self
+        Query Params:  <none>
+        Body (JSON):   {
+                         "data": {
+                           "type": "nodes", # required
+                           "attributes": {
+                             "title":       {title},         # mandatory
+                             "category":    {category},      # mandatory
+                             "description": {description},   # optional
+                             "tags":        [{tag1}, {tag2}] # optional
+                           }
+                         }
+                       }
+        Success:       201 CREATED + node representation
+
+    New nodes are created by issuing a POST request to this endpoint.  The `title` and `category` fields are
+    mandatory. `category` must be one of the [permitted node categories](/v2/#osf-node-categories).  All other fields
+    not listed above will be ignored.  If the node creation is successful the API will return a 201 response with the
+    respresentation of the new node in the body.  For the new node's canonical URL, see the `links.self` field of the
+    response.
+
+    ##Query Params
+
+    + `page=<Int>` -- page number of results to view, default 1
+
+    + `filter[<fieldname>]=<Str>` -- fields and values to filter the search results on.
+
```

Comment:
We should also note the 
```related_counts=true```
param will add counts to related fields. This might belong in that general part of the docs.